documentation/CAMARA-Security-Interoperability.md

```diff
@@ -258,8 +258,9 @@ This document does not mandate a particular PPID algorithm to be used.
 
 ## Client Authentication
 
-This CAMARA document allows **one** client authentication method, `private_key_jwt`, as defined in OIDC
-[OIDC Client Authentication](https://openid.net/specs/openid-connect-core-1_0.html#ClientAuthentication)
+The API consumer MUST authenticate with the authorisation server using `private_key_jwt`, as specified in [OIDC Client Authentication](https://openid.net/specs/openid-connect-core-1_0.html#ClientAuthentication). In addition to the mandated claims, the signed JWT SHOULD also include the `iat` (issued at) claim.
```

Comment:
```suggestion
This CAMARA document allows **one** client authentication method, `private_key_jwt`, as defined in OIDC.
```

Reply:
@AxelNennker
There is no point in including "The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119" in the document and then not using these keywords.

If an API consumer must do something, then they MUST do it. Stating "This CAMARA document allows" suggests there might be another CAMARA document that allows something different.